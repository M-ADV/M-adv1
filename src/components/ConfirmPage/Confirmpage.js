import React from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faCheckCircle } from '@fortawesome/free-solid-svg-icons';

const Confirmpage = () => {
  return (
    <>
      <div
        className="card bg-dark mt-5 "
        style={{ width: '45rem', marginLeft: '20rem' }}
      >
        <h3 className="card-header text-light Cardtop">
          Congratulations{' '}
          <FontAwesomeIcon
            className="fa-beat-fade"
            icon={faCheckCircle}
          ></FontAwesomeIcon>{' '}
        </h3>

        <div
          className="card-body"
          style={{ backgroundColor: 'rgb(169 47 51)' }}
        >
          <h4 className="card-text text-light">
            Thanks for choosing usðŸ˜‡ <br />
            Our executive will reach you soon.... <br />
            Please Be ready with your all documents which need to be submitted.
            <br />
            Thank you!
          </h4>
        </div>
      </div>
    </>
  );
};

export default Confirmpage;
