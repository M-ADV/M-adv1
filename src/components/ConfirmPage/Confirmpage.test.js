import { render, screen } from "@testing-library/react";
import { renderComponent } from "../../utils/test_helpers";
import Confirmpage from "./Confirmpage";

describe("Confirmpage", () => {
  test("Initial render", () => {
    render(renderComponent(<Confirmpage />));
    expect(
      screen.getByRole("heading", { name: "Congratulations" })
    ).toBeInTheDocument();
  });
  test("Matching the snapshot of the navbar", async () => {
    const { asFragment } = render(renderComponent(<Confirmpage />));
    expect(asFragment()).toMatchSnapshot();
  });
});
