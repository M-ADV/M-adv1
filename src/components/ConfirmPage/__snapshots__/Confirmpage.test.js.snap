// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Confirmpage Matching the snapshot of the navbar 1`] = `
<DocumentFragment>
  <div
    class="card bg-dark mt-5 "
    style="width: 45rem; margin-left: 20rem;"
  >
    <h3
      class="card-header text-light Cardtop"
    >
      Congratulations 
      <svg
        aria-hidden="true"
        class="svg-inline--fa fa-circle-check fa-beat-fade"
        data-icon="circle-check"
        data-prefix="fas"
        focusable="false"
        role="img"
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M256 512c141.4 0 256-114.6 256-256S397.4 0 256 0S0 114.6 0 256S114.6 512 256 512zM369 209L241 337c-9.4 9.4-24.6 9.4-33.9 0l-64-64c-9.4-9.4-9.4-24.6 0-33.9s24.6-9.4 33.9 0l47 47L335 175c9.4-9.4 24.6-9.4 33.9 0s9.4 24.6 0 33.9z"
          fill="currentColor"
        />
      </svg>
       
    </h3>
    <div
      class="card-body"
    >
      <h4
        class="card-text text-light"
      >
        Thanks for choosing usðŸ˜‡ 
        <br />
        Our executive will reach you soon.... 
        <br />
        Please Be ready with your all documents which need to be submitted.
        <br />
        Thank you!
      </h4>
    </div>
  </div>
</DocumentFragment>
`;
